### ISSUE TITLE
Config errors

### ISSUE THREAD

**Issue by tygryk** [laacz/xls-reader#1]
Hi,

documentaton says 

$book = new laacz\XLSReader\Book(file_get_content('workbook.xls'));

did not work for me. What worked was 

$book = new laacz\XLSParser\Book(file_get_content('workbook.xls'));

When called, the following error:

Constant IDENTIFIER_OLE already defined

```
1. in blah-blah/vendor/laacz/xls-parser/src/Book.php at line 48
```

Good luck!

**Comment by laacz** [comment]
Hello! I believe you've included this library twice or have a constant definition in some other place in your code tree.

**Comment by tygryk** [comment]
Hi,

You don't hear me, bro. There is
1) error in your documentation
2) error in your code

I ended up using another extension.

**Comment by laacz** [comment]
Oh, sorry. I misread you. Fixed example with 0f4ee14e4bdc050ad0c844a74629cbc042523796
